### PR TITLE
fix: double require problem

### PR DESCRIPTION
**Pull request by rishabh3112**
**What kind of change does this PR introduce?**
Fix

**Did you add tests for your changes?**
N/A

**If relevant, did you update the documentation?**
N/A

**Summary**
Fix double require problem. Described as in https://github.com/webpack/webpack-dev-server/pull/3219

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

**Comment by rishabh3112** [comment]
Not working on windows 😞 . @webpack/cli-team can you help?

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [windows-latest]
```
Comment (evenstensberg) [inline .github/workflows/nodejs.yml:56]: Why did you remove macos and ubuntu?

File: packages/webpack-cli/lib/utils/package-exists.js [hunk 2]
```diff
@@ -1,7 +1,24 @@
...
+        }`;
+
+        const { stdout: rootPath } = await execa('node', ['-e', command], {
+            cwd: path.resolve(process.cwd()),
+            reject: false,
+            stdio: 'inherit',
+            maxBuffer: Infinity,
+        });
+
+        console.log(rootPath);
```
Comment (evenstensberg) [inline packages/webpack-cli/lib/utils/package-exists.js:19]: console.log

File: .github/workflows/nodejs.yml (hunk 1 above)
Comment (anshumanv) [reply .github/workflows/nodejs.yml:56]: maybe debug

**Comment by rishabh3112** [comment]
@webpack/cli-team ready for review now!!

**Comment by rishabh3112** [comment]
Closing this in favour of #2716
